doc/source/parameters/cfd/boundary_conditions_cfd.rst

```diff
@@ -0,0 +1,48 @@
+Boundary conditions - CFD
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This subsection's defines the boundary conditions associated with fluid dynamics physics. Lethe supports slip, no-slip, periodic, and dirichlet boundary conditions which are set from arbitrary functions. These functions can be used to define all sort of steady-state and transient velocity boundary conditions such as rotating walls.
```

Comment:
It's me being me again. 
I would't say arbitrary Dirichlet. I would rather stick to slip, no-slip, periodic, velocity inlet,  and outflow.  
Isn't a slip BC a mix between Dirichlet for the normal component of the velocity and Newman for the total stress tensor? Can we define a Dirichlet for pressure now? 
Might be wrong here